Readme.md

```diff
@@ -1,6 +1,6 @@
 # Codeship Docker Infrastructure Tutorial
 
-Please see our Codeship Docker Infrastructure [Getting Started](https://codeship.com/documentation/docker/getting-started/) tutorial for the full text as well as installation instructions for the CLI.
+Please see our Codeship Docker Infrastructure [Getting Started](https://codeship.com/documentation/docker/getting-started/getting-started) tutorial for the full text as well as installation instructions for the CLI.
```

Comment:
Could you update the link to https://documentation.codeship.com/docker/getting-started/getting-started/